front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/ServiceAccountBucketDAO.java

```diff
@@ -41,7 +41,6 @@ public void update(String id, ServiceAccount permission) {
 
   private ServiceAccount upsert(String id, ServiceAccount permission) {
     permission.setName(id);
-    permission.setLastModified(System.currentTimeMillis());
```

Comment:
These are unnecessary calls as `lastModified` is determined by the object timestamp in GCS (and S3).